src/network/clientpackethandler.cpp

```diff
@@ -1116,6 +1117,8 @@ void Client::handleCommand_HudAdd(NetworkPacket* pkt)
 		*pkt >> size;
 	} catch(SerializationError &e) {};
 
+	*pkt >> font_size;
```

Comment:
move that into the try catch block

Reply:
Plus the old try/catch blocks can be removed, as these params are guaranteed to be sent after the backwards compat drop.